src/isomorphic/classic/element/ReactElementValidator.js

```diff
@@ -19,6 +19,12 @@
 var ReactCurrentOwner = require('ReactCurrentOwner');
 var ReactElement = require('ReactElement');
 
+const REACT_FRAGMENT_TYPE =
+  (typeof Symbol === 'function' &&
+    Symbol.for &&
+    Symbol.for('react.fragment')) ||
+  0xad9c;
+
```

Comment:
We should move this into the `__DEV__` block because I don't think it gets DCE otherwise since `Symbol.for()` could be a stateful function.

Reply:
Is `ReactElementValidator` not used when `__DEV__` is off?

Reply:
Nope.